articles/tutorials/step-up-authentication.md

```diff
@@ -5,71 +5,71 @@ description: You can add step-up authentication to your app with Authentication
 
-For example, *Fabrikam's Intranet* can require users to authenticate with their username and password to access customer data. However, a request for access to employee data (which may contain sensitive salary information) can trigger a stronger authentication mechanism like [multifactor authentication](/multifactor-authentication).
-
-You can add step-up authentication to your app with Auth0's extensible multifactor authentication support. Your app can verify that the user has logged in using multifactor authentication and, if not, require the user to step-up to access certain resources.
+If you have an app called Fabrikam that allows users to access different types of resources, you can vary the level of authentication required. For example, you might require a user to log into your portal using a username/password combination, but if they request access to Employees page, which contains sensitive details such as salary information, your app will ask your user to complete the [multifactor authentication](/multifactor-authentication) process before it grants the user their request.
 
 ![](/media/articles/step-up-authentication/flow.png)
 
-## Step-Up Authentication with ACR
+## Key Terms When Using Step-Up Authentication
 
-There are three core concepts used when addressing authentication level at Auth0.
+The following terminology is important when discussing step-up authentication using Auth0.
 
-* `acr`- Authentication Context Class Reference: is a string used to specify the 'class' of authentication that was performed on the current session. Look to [Authentication Context Class Reference](http://openid.net/specs/openid-connect-core-1_0.html) page for more detail and specific policies. Currently, Auth0 utilizes the 'Multi-Factor Authentication' policy, `http://schemas.openid.net/pape/policies/2007/06/multi-factor`.
+* Authentication Context Class Reference `acr`: string used to specify the authentication class performed on the current session. See `acr` under [ID Token](http://openid.net/specs/openid-connect-core-1_0.html#IDToken). Currently, Auth0 utilizes the [Multi-Factor Authentication policy](http://schemas.openid.net/pape/policies/2007/06/multi-factor).
 
-* `amr`- Authentication Methods References: is a JSON case senstitive string list of methods that were used to authenticate the current session.  For instance, values might indicate that both password and OTP authentication methods were used.  See the [Authentication Methods References](http://openid.net/specs/openid-connect-core-1_0.html) page for more details.
+* Authentication Methods References `amr`: JSON array of strings listing the methods used to authenticate the current session. For example, the `amr` might indicate that the current session was authenticated using a username/password.  See `amr` under [ID Token](http://openid.net/specs/openid-connect-core-1_0.html#IDToken).
```

Comment:
`amr` is looking very good, but I did not understand what `acr` is. Maybe explain it a bit more?